contracts/ts/genMaciState.ts

```diff
@@ -237,12 +235,6 @@ export const genMaciStateFromContract = async (
         break;
       }
 
-      // ensure that the message root is correct (i.e. all messages have been published offchain)
-      case action.type === "MergeMessageAq": {
-        assert(maciState.polls.get(pollId)?.messageTree.root.toString() === action.data.messageRoot?.toString());
-        break;
-      }
-
       default:
         break;
     }
```

Comment:
we need to check that we pulled all messages, so let's take the final chain Hash value from the Poll contract and check that it matches the one in the Poll.ts object